api/config/config_prod.json

```diff
@@ -12,7 +12,7 @@
     "xAppIdValue": "AoU-RW",
     "timeoutInSeconds": 40,
     "lenientTimeoutInSeconds": 180,
-    "jupyterDockerImage": "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.7",
```

Comment:
shall we separate prod in a follow up just in case we found anything surprising?

Reply:
good idea. Reverting preprod and prod